scripts/plot_run_duration_over_time.py

```diff
@@ -22,28 +22,34 @@
 for benchmark_json in data_json:
 	name = benchmark_json['name']
 	benchmark_names.append(name)
-	for run_json in benchmark_json['runs']:
-		begin = run_json['begin']
-		duration = run_json['duration']
-		data.append({'name': name, 'begin': begin / 1e9, 'duration': duration})
+	for run_json in benchmark_json['successful_runs']:
+		data.append({'name': name, 'begin': run_json['begin'] / 1e9, 'duration': run_json['duration'], 'success': True})
+	for run_json in benchmark_json['unsuccessful_runs']:
```

Comment:
I'm not really happy with the code duplication here, but I have no idea how to avoid it without making everything more verbose.

Reply:
I am sure we can handle these two duplicated lines